xapian-core/weight/bm25weight.cc

```diff
@@ -35,11 +35,23 @@ using namespace std;
 
 namespace Xapian {
 
+BM25Weight::BM25Weight(bool& bm25+)
```

Comment:
What name do you suggest for this parameter? I was also thinking of `bm25_plus` earlier.